# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxFontEntry.h
# Commit: 5d1de3ed837b
# Full Hash: 5d1de3ed837b7c7c7a2f811a7bb27473d981dca0
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-08-17 15:40:28
# Regressor Bug: 1740530
# File Overlap Count: 1
# Description:
#   Bug 1740530 - patch 3 - Replace GetColorGlyphLayers and the layer-rendering code in gfxFont with a zero-copy implementation in COLRFonts. r=gfx-reviewers,lsalzman
#   
#   Depends on D152037
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152038
# ==============================================================================

diff -r 13ae409d093a -r 5d1de3ed837b gfx/thebes/gfxFontEntry.h
--- a/gfx/thebes/gfxFontEntry.h	Wed Aug 17 10:59:33 2022 +0000
+++ b/gfx/thebes/gfxFontEntry.h	Wed Aug 17 10:59:34 2022 +0000
@@ -58,9 +58,6 @@
 struct Face;
 struct Family;
 }  // namespace fontlist
-namespace gfx {
-struct DeviceColor;
-}
 }  // namespace mozilla
 
 typedef struct gr_face gr_face;
